docs/asdf/extending/extensions.rst

```diff
@@ -253,18 +253,20 @@ We'll assume that method is located in the module ``asdf_foo_extension.integrati
 Next, in the package's ``pyproject.toml``, define a ``[project.entry-points]`` section (or ``[options.entry_points]`` in
 ``setup.cfg``) that identifies the method as an ``asdf.extensions`` entry point:
 
-.. code-block:: toml
+.. tab:: pyproject.toml
 
-    # pyproject.toml
-    [project.entry-points]
-    'asdf.extensions' = { asdf_foo_extension = 'asdf_foo_extension.integration:get_extensions' }
+    .. code-block:: toml
 
-.. code-block:: ini
+        [project.entry-points]
+        'asdf.extensions' = { asdf_foo_extension = 'asdf_foo_extension.integration:get_extensions' }
```

Comment:
@zacharyburnett should we multi-line these?

Reply:
for reference, multi-line inline tables will be supported when https://github.com/toml-lang/toml/pull/904 is merged and released. However, we could do this:
```toml
[project.entry-points.'asdf.extensions']
asdf_foo_extension = 'asdf_foo_extension.integration:get_extensions' 
```
though this might be unwieldy and confusing for users

Reply:
I say we leave it as is. It is not bad, just in the docs it requires horizontal scrolling.